src/runtime/objmodel.cpp

```diff
@@ -2520,12 +2524,12 @@ Box* callFunc(BoxedFunctionBase* func, CallRewriteArgs* rewrite_args, ArgPassSpe
         getArg(i + positional_to_positional, oarg1, oarg2, oarg3, oargs) = varargs[i];
     }
 
-    std::vector<bool> params_filled(num_output_args, false);
+    llvm::SmallVector<bool, 8> params_filled(num_output_args);
     for (int i = 0; i < positional_to_positional + varargs_to_positional; i++) {
         params_filled[i] = true;
     }
 
-    std::vector<Box*, StlCompatAllocator<Box*>> unused_positional;
+    llvm::SmallVector<Box*, 8> unused_positional;
```

Comment:
Hmm I feel iffy about this one... I guess it's true that arg1/arg2/arg3/args are all referenced lower than here which I think will force the values to stay tracked, but I'm not sure if we should rely on that.